<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDownloadDialog</class>
 <widget class="QDialog" name="AddDownloadDialog">
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="viewStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageValidate">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="verticalSpacing">
          <number>12</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblMediaUrl"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtMediaUrl"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblAuthenticate"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cmbAuthenticate"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblUsername"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="txtUsername"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblPassword"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="txtPassword"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnValidate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset theme="applications-internet"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSpinner">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSingle">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <item row="7" column="0" colspan="2">
          <widget class="QLineEdit" name="txtFilenameSingle"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lblQualitySingle"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblFileTypeSingle"/>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lblPreferAV1Single"/>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="lblTimeFrameEndSingle"/>
         </item>
         <item row="9" column="0">
          <widget class="QLineEdit" name="txtTimeFrameStartSingle"/>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="btnSelectSaveFolderSingle">
           <property name="icon">
            <iconset theme="folder-open"/>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="lblLimitSpeedSingle"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cmbQualitySingle"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblDownloadSubtitlesSingle"/>
         </item>
         <item row="9" column="2">
          <widget class="QCheckBox" name="chkLimitSpeedSingle"/>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLineEdit" name="txtSaveFolderSingle"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblFilenameSingle"/>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="cmbFileTypeSingle"/>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="txtTimeFrameEndSingle"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="lblSplitChaptersSingle"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="chkDownloadSubtitlesSingle"/>
         </item>
         <item row="10" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="cmbAudioLanguageSingle"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblSaveFolderSingle"/>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="chkPreferAV1Single"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="lblTimeFrameStartSingle"/>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="btnRevertFilenameSingle">
           <property name="icon">
            <iconset theme="edit-undo"/>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="chkSplitChaptersSingle"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lblAudioLanguageSingle"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnDownloadSingle">
         <property name="icon">
          <iconset theme="emblem-downloads"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePlaylist">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblFileTypePlaylist"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblSplitChaptersPlaylist"/>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="cmbFileTypePlaylist"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblItemsPlaylist"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblSaveFolderPlaylist"/>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="chkLimitSpeedPlaylist"/>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="chkDownloadSubtitlesPlaylist"/>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lblLimitSpeedPlaylist"/>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="chkNumberTitlesPlaylist"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="chkSplitChaptersPlaylist"/>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="chkPreferAV1Playlist"/>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="btnSelectSaveFolderPlaylist">
           <property name="icon">
            <iconset theme="folder-open"/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lblDownloadSubtitlesPlaylist"/>
         </item>
         <item row="7" column="0" colspan="3">
          <widget class="QTableWidget" name="tblItemsPlaylist">
           <property name="columnCount">
            <number>3</number>
           </property>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="lblNumberTitlesPlaylist"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lblPreferAV1Playlist"/>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLineEdit" name="txtSaveFolderPlaylist">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnDownloadPlaylist">
         <property name="icon">
          <iconset theme="emblem-downloads"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
